back/cmd/openbsrv/main.go

```diff
@@ -7,17 +7,32 @@ import (
 	"path"
 
 	"github.com/codemodus/sigmon/v2"
+
+	"github.com/OpenEugene/openboard/back/internal/dbg"
+	"github.com/OpenEugene/openboard/back/internal/log"
 )
 
 func main() {
-	if err := run(); err != nil {
+	config := log.Config{
+		Err: log.Output{
+			Out:    os.Stderr,
```

Comment:
Technically, this wouldn't be stderr. stderr is for application-level failures. For instance, if the application cannot connect to the database and halts, we would report this to stderr. The "error log" here is really for messages we want to log that are errors, but that are just being stored and do not halt the application. For instance, if a client posts a request containing some data that causes some interesting edge-case failure.

Reply:
For me, this brings into question whether this is a race condition since two different log.logger instances may write to the same file. It's probably worth looking into it briefly - please file a separate issue to investigate.

Reply:
Issue #159 has been filed.